services/tenant-ui/frontend/src/components/holder/Credentials.vue

```diff
@@ -46,17 +74,44 @@ import { useToast } from 'vue-toastification';
 
 import { useHolderStore } from '../../store';
 import { storeToRefs } from 'pinia';
+import { useConfirm } from 'primevue/useconfirm';
 
 import { formatDateLong } from '@/helpers';
 
 const toast = useToast();
+const confirm = useConfirm();
 
 const holderStore = useHolderStore();
 // use the loading state from the store to disable the button...
 const { loading, credentials, selectedCredential } = storeToRefs(
   useHolderStore()
 );
 
+const acceptOffer = (event: any, data: any) => {
+  holderStore.acceptCredentialOffer(data.holder_credential_id).then(() => {
+    loadTable();
```

Comment:
this isn't the same pattern as used elsewhere for having the table refreshed on add/delete/etc. for good or ill, they should be the same for now.

Reply:
I would support calling the refresh function after these as well, the table should do it's thing after you accept/delete and then you'd see the updated entry.

Reply:
It appears the existing pattern is for the store operation to update the related affected objects. 

So the table get's update because the store updates itself, rather than leaving it to the component to manage the refresh. Just something i missed, however that store code itself could use some simplification\.

Reply:
yeah, we don't really know what we want to do with the store and data anyway. I'm not super convinced we need to store anything but the token/creds for the tenant.  the store design allows for a bunch of different ways of interacting with the async stuff, like the same function can be used as we are using it, and also support subscribing to 'onAction'. this pattern has emerged, so we should strip out the extra stuff to support subscriptions...